packages/react-components/react-drawer/library/src/components/InlineDrawer/renderInlineDrawer.tsx

```diff
@@ -3,21 +3,21 @@
 import { assertSlots } from '@fluentui/react-utilities';
 import { DrawerContextValue, DrawerProvider } from '../../contexts/drawerContext';
 
+import { InlineDrawerMotion } from '../../shared/drawerMotions';
 import type { InlineDrawerState, InlineDrawerSlots } from './InlineDrawer.types';
 
 /**
  * Render the final JSX of InlineDrawer
  */
 export const renderInlineDrawer_unstable = (state: InlineDrawerState, contextValue: DrawerContextValue) => {
-  if (!state.motion.canRender) {
-    return null;
-  }
-
   assertSlots<InlineDrawerSlots>(state);
+  const { open, size, position } = state;
 
   return (
     <DrawerProvider value={contextValue}>
-      <state.root />
+      <InlineDrawerMotion position={position} size={size} visible={open} unmountOnExit>
```

Comment:
How would the overrides for animation work in this case? Should that be a configurable slot that users can pass to override this motion component?

Reply:
Then plan is to have a slot as you mentioned, it's not implemented yet: #30700.

Reply:
This could be a pattern that we can use moving forward. If we opt for using slots as our override solution, all components that ship with animation by default (Drawer, Dialog, Nav, more?) could have the same Slot name for motion.

``` html
<OverlayDrawer motion={CustomDrawerMotion} />
<DialogSurface motion={CustomDialogSurfaceMotion} />
```

We can discuss that another time, but just registering here the idea. 🙂

Reply:
@marcosmoura I added slots for `Dialog` in https://github.com/microsoft/fluentui/pull/31132, and modified this PR to add slots for `*Drawer` components 👍  Please check, I also updated stories 🐱